[2024-03-17T08:03:41.242+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: study_case_etl_cloud_storage_tiga.bigquery_external_table_task manual__2024-03-17T08:03:27.270408+00:00 [queued]>
[2024-03-17T08:03:41.251+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: study_case_etl_cloud_storage_tiga.bigquery_external_table_task manual__2024-03-17T08:03:27.270408+00:00 [queued]>
[2024-03-17T08:03:41.252+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-17T08:03:41.264+0000] {taskinstance.py:2214} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2024-03-17 08:03:27.270408+00:00
[2024-03-17T08:03:41.269+0000] {standard_task_runner.py:60} INFO - Started process 7911 to run task
[2024-03-17T08:03:41.273+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'study_case_etl_cloud_storage_tiga', 'bigquery_external_table_task', 'manual__2024-03-17T08:03:27.270408+00:00', '--job-id', '1014', '--raw', '--subdir', 'DAGS_FOLDER/etl_day_bigquery/study_case_etl_cloud_bigquery.py', '--cfg-path', '/tmp/tmpe8o0ijy_']
[2024-03-17T08:03:41.276+0000] {standard_task_runner.py:88} INFO - Job 1014: Subtask bigquery_external_table_task
[2024-03-17T08:03:41.314+0000] {task_command.py:423} INFO - Running <TaskInstance: study_case_etl_cloud_storage_tiga.bigquery_external_table_task manual__2024-03-17T08:03:27.270408+00:00 [running]> on host 1d5b76b26ba5
[2024-03-17T08:03:41.389+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Agus mahari' AIRFLOW_CTX_DAG_ID='study_case_etl_cloud_storage_tiga' AIRFLOW_CTX_TASK_ID='bigquery_external_table_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-17T08:03:27.270408+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-17T08:03:27.270408+00:00'
[2024-03-17T08:03:41.391+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-03-17T08:03:41.392+0000] {connection.py:269} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-03-17T08:03:41.393+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-03-17T08:03:42.204+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=study_case_etl_cloud_storage_tiga, task_id=bigquery_external_table_task, execution_date=20240317T080327, start_date=20240317T080341, end_date=20240317T080342
[2024-03-17T08:03:42.230+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-17T08:03:42.252+0000] {taskinstance.py:3309} INFO - 2 downstream tasks scheduled from follow-on schedule check
